### ISSUE TITLE
HandlingData error for custom vehicles

### ISSUE THREAD

**Issue by mrgharabaghi** [altmp/altv-issues#1173]
### Description of the problem

`alt.HandlingData()` doesn't return the `HandlingData` for custom vehicles.

### Reproduction steps

```js
let handling = new alt.HandlingData(alt.hash('pvic')); // pvic -> Handling ID
// Error: model doesn't exist

alt.HandlingData.getForHandlingName(alt.hash('pvic')); // pvic -> Handling ID
// Error: model doesn't exist
```

### Expected behaviour

Return the `HandlingData` for custom vehicles.

### Operating system

Windows 10

### Version

release/6.4

**Comment by xxshady** [comment]
Handling API doesn't working not only for custom vehicles, but also for some vanilla too, e.g. blista2, blista

**Comment by zziger** [comment]
Fixed in `dev/15.0-dev844`
